### ISSUE TITLE
How to add to sails.hooks.blueprints?

### ISSUE THREAD

**Issue by JemiloII** [balderdashy/sails#2045]
So I created a policy and modified ok.js in sailsjs to use the jsonapi.org standard.

Currently I'm requiring config/blueprints in both files. I thought I could do sails.hooks.blueprints to see jsonApi: true and jsonApiPluralize: false values. But it doesn't recognize the new values in my blueprints.

Working on extending sails to help everyone out. Frameworks like ember.js use the jsonapi, and I figure there are many more. Thanks very much appreciated!

**Comment by CWyrtzen** [comment]
Did this get cleared up?

**Comment by sailsbot** [comment]
Thanks for posting, @JemiloII.  I'm a repo bot-- nice to meet you!

It has been 60 days since there have been any updates or new comments on this page.  If this issue has been resolved, feel free to disregard the rest of this message.  On the other hand, if you are still waiting on a patch, please:

  + review our [contribution guide](https://github.com/balderdashy/sails/blob/master/CONTRIBUTING.md) to make sure this submission meets our criteria (only _verified bugs_ with documented features, please;  no questions, commentary, or bug reports about undocumented features or unofficial plugins)
  + create a new issue with the latest information, including updated version details with error messages, failing tests, and a link back to [the original issue]().  This allows GitHub to automatically create a back-reference for future visitors arriving from search engines.

Thanks so much for your help!
